sleap_io/model/skeleton.py

```diff
@@ -37,22 +39,129 @@ class Edge:
     source: Node
     destination: Node
 
+    def __getitem__(self, idx):
+        if idx == 0:
+            return self.source
+        elif idx == 1:
+            return self.destination
+        else:
+            raise ValueError("Edge only has 2 nodes (source and destination).")
 
-@define(auto_attribs=True)
+
+@define
+class Symmetry:
+    """A relationship between a pair of nodes denoting their left/right pairing.
+
+    Attributes:
+        nodes: A set of two `Node`s.
+    """
+
+    nodes: set[Node] = field(converter=set, validator=lambda _, __, val: len(val) == 2)
+
+
+@define
 class Skeleton:
     """A description of a set of landmark types and connections between them.
 
     Skeletons are represented by a directed graph composed of a set of `Node`s (landmark
     types such as body parts) and `Edge`s (connections between parts).
 
     Attributes:
-        nodes: A list of `Node`s.
-        edges: A list of `Edge`s.
+        nodes: A list of `Node`s. May be specified as a list of strings to create new
```

Comment:
I could be wrong, but it seems like sphinx has some problems rendering the link to the class if we add a non-whitespace character immediately following the closing backtick.

Reply:
Yep, we might be able to [use an escape](https://stackoverflow.com/questions/29837793/restructuredtext-backquote-text-immediate-followed-by-other-letters) or maybe just reconfigure sphinx to get around this in the future.